app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -48,10 +40,10 @@
     .govuk-breadcrumbs__link:after {
```

Comment:
Related - I think it's possible to remove line 32: `margin: 0` on `.govuk-breadcrumbs__list-item:before`. Testing it locally and it's already set.

Reply:
Indeed, L32 is not needed. Regarding the other comment, when rebasing I re-added the `position: relative` mentioned in my comment above 🤦 . Removed it again – it should look as in my screen capture again.